public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Mel's hack - the missing bits"
+author: "(David *)Frenkiel"
+date: Thu Jun 23 2022 01:14:31 GMT+0300
+---
+
+### A private _Mel moment_
+
+        When the light went on it nearly blinded me. 
+
+This line, in The Story of Mel, precedes the author's description of Mel's hack, the almost criminally resourceful implementation of a finite loop with no exit condition. These words never fail to give me goose bumps. They reflect a rare "Ah!" moment, where a seemingly random collection of facts suddenly fall into a coherent logical structure.
```

Comment:
Not for current version, but should be added as a ticket for future improvement: Add a link to the line's anchor. (Demands a bugfix)